awscli/customizations/cliinput.py

```diff
@@ -145,7 +145,9 @@ class CliInputYAMLArgument(CliInputArgument):
     }
 
     def _load_parameters(self, arg_value):
+        yaml = YAML(typ='safe')
```

Comment:
So one thing that it would be good to follow up on is going through the ruamel source code, the deprecation warnings seem to recommend setting `pure=True` when instantiating the `YAML` class when migrating away from the module level `load()` method. For this upgrade, we should try to minimize the changes so if we were using the pure python abstractions before, we should continue to use those instead of the C-based ones. Would it be possible to look into this to see if it changes in upgrading to this new usage?